test/native/setup.js

```diff
@@ -177,16 +177,28 @@ jest.mock( 'react-native/Libraries/LayoutAnimation/LayoutAnimation' );
 jest.mock(
 	'react-native/Libraries/Components/AccessibilityInfo/AccessibilityInfo',
 	() => ( {
-		addEventListener: jest.fn(),
-		announceForAccessibility: jest.fn(),
-		removeEventListener: jest.fn(),
-		isScreenReaderEnabled: jest.fn( () => Promise.resolve( false ) ),
-		fetch: jest.fn( () => ( {
-			done: jest.fn(),
-		} ) ),
+		__esModule: true,
+		default: {
+			addEventListener: jest.fn( () => ( { remove: jest.fn() } ) ),
+			announceForAccessibility: jest.fn(),
+			isBoldTextEnabled: jest.fn(),
+			isGrayscaleEnabled: jest.fn(),
+			isInvertColorsEnabled: jest.fn(),
+			isReduceMotionEnabled: jest.fn(),
+			isReduceTransparencyEnabled: jest.fn(),
+			isScreenReaderEnabled: jest.fn( () => Promise.resolve( false ) ),
+			removeEventListener: jest.fn(),
+			setAccessibilityFocus: jest.fn(),
+			sendAccessibilityEvent_unstable: jest.fn(),
+			getRecommendedTimeoutMillis: jest.fn(),
+		},
 	} )
 );
-jest.mock( 'react-native/Libraries/Components/Clipboard/Clipboard', () => ( {
+
+// The mock provided by the package itself does not appear to work correctly.
+// Specifically, the mock provides a named export, where the module itself uses
+// a default export.
+jest.mock( '@react-native-clipboard/clipboard', () => ( {
```

Comment:
Hey David, looks like there's still [this other mock of the Clipboard lib](https://github.com/WordPress/gutenberg/blob/c3307b48de08df88851d9d16f09d764c350ce1e3/test/native/setup.js#L157-L166), not sure if keeping both in is intentional?

Reply:
Good catch! I removed this in https://github.com/WordPress/gutenberg/pull/36328/commits/6d64599f8fe1b1eac4d3e0f415e7e88ff8a60e2d.